ironfish-rust-nodejs/src/lib.rs

```diff
@@ -25,8 +25,7 @@ pub struct Key {
 
 #[napi]
 pub fn generate_key() -> Key {
-    let hasher = sapling_bls12::SAPLING.clone();
-    let sapling_key = sapling_bls12::Key::generate_key(hasher);
+    let sapling_key = sapling_bls12::Key::generate_key();
```

Comment:
We're using `generateKey()` in a few places on the TS side in order to trigger the loading of the Sapling parameters, which takes a couple seconds. Notably, in the workers, so that the first time we e.g. generate a miner's fee transaction, it's as fast as possible.

Since this function no longer uses the global SAPLING instance, we'll probably need a different way of doing this. It'd probably be better to have some explicit function like `loadSaplingParameters` rather than implicitly relying on `generateKey` to do it.